tests/mocha/event_block_field_intermediate_change_test.js

```diff
@@ -35,4 +35,38 @@ suite('Field Intermediate Change Event', function () {
       chai.assert.deepEqual(newEvent, origEvent);
     });
   });
+
+  suite('Change Value', function () {
+    test("runs with forward change block's value to new value", function () {
```

Comment:
Can you change these to be of the form "this tests that X" where X becomes the title of the test?

E.g. "running forward changes the block's value to the new value", and "running backward changes the block's value to the old value".